native/src/components/List.tsx

```diff
@@ -44,11 +44,7 @@ const List = <T,>({
     }
     onEndReached={onEndReached}
     showsVerticalScrollIndicator={false}
-    contentContainerStyle={{
-      flexGrow: 1,
-
-      paddingHorizontal: 10,
-    }}
+    contentContainerStyle={{ flex: 1 }}
```

Comment:
@steffenkleinle Why have you changed this? This causes this bug: https://github.com/digitalfabrik/integreat-app/issues/2660

I reverted here: Feel free to comment if this breaks something you fixed.

Reply:
Hmm, I think I had some layout problems, but @f1sh1918 suggestions already fixed them. So yes, feel free to revert.